apps/docs/components/edit-feedback.tsx

```diff
@@ -0,0 +1,37 @@
+type MinimalPost = {
+  title: string
+  slug: string
+}
+
+type Props = {
+  post: MinimalPost
+  contentPath: string
+}
+
+export default function EditFeedbackLinks({ post, contentPath }: Props) {
```

Comment:
Let's just pass the post title as a prop, rather than the whole post, since that's all we need here